### PR TITLE
rpc sandbox supporting treasury factory

### LINKED ISSUE #193: Deploy Treasury factory and sputnik-dao in RPC sandbox
### Background

To be able to create a Playwright test for the self creation flow, and then to be able to test user behaviour efficiently, the RPC sandbox should have all the necessary contracts deployed so that it is possible to write a full end-2-end Playwright test.

### User Story

As a user of the self-creation flow
I need that to know that creating a Treasury portal using the self creation is being constantly tested, and that all known user behaviour scenarios are tested
So that I can be sure that creation of a Treasury Portal with the needed sputnik-dao contract is successful, and that I would not risk loosing funds because of an unsuccessful creation process

### Acceptance Criteria

- Verify that the RPC sandbox has all contracts deployed needed to write a full end 2 end test for the self creation flow
- Verify that the Playwright test can be implemented without mocking transactions or blockchain storage
- 

### Resources & Additional Notes

_No response_

### Priority

🔴 P0 : Critical

### PR DESCRIPTION
**Pull request by petersalomonsen**
This PR adds the `treasury-factory.near` contract and the dependencies `social.near` and `near` contracts to the RPC sandbox which can be used for Playwright tests and for experimenting in the [playground](https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/blob/feat/sandbox-treasury-factory/sandboxrpc/README.md).

The file [sandboxrpc/playground/treasuryfactory.js](https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/blob/feat/sandbox-treasury-factory/sandboxrpc/playground/treasuryfactory.js) shows how to use the treasury factory with the sandbox. You can run this file with nodejs and test various ways of interaction.

Also there is a [Playwright test](https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/blob/feat/sandbox-treasury-factory/playwright-tests/tests/treasury-factory/treasury-factory.spec.js) that shows how to use it in Playwright and make the needed assertions that a treasury instance was successfully created. This can be used as a starting point for creating the full E2E test in https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/pull/191

Broken test fixes:

- "Should update existing member permissions" ( the locator for finding the delete button was not specific enough ).
- "Should whitelist staking pool and create stake delegation request" ( also made the locator more specific ).

resolves #193

### CODE REVIEW

**Comment by petersalomonsen** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@petersalomonsen Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
Great job! Usain Bolt jealous of your results!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/petersalomonsen?type=bot&pr=NEAR-DevHub/neardevhub-treasury-dashboard/194&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/petersalomonsen?type=bot&pr=NEAR-DevHub/neardevhub-treasury-dashboard/194&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/petersalomonsen)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/petersalomonsen)

For maintainers:
- Score pull requests that participate in the Race of Sloths and receive a reward
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

File: .github/workflows/test.yml
Comment (Megha-Dev-19) [inline .github/workflows/test.yml:59]: I don't think we need matrix if we are running it only for one project

**Review by Megha-Dev-19 — APPROVED** [review]

**Comment by Megha-Dev-19** [comment]
@race-of-sloths score 8

**Pull request merged**
